content/blog/2024/06/2024-06-13-thanks-mirror-sponsors.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Thank you supporters of Jenkins providing download Mirrors"
+tags:
+- community
+- infrastructure
+- sponsors
+- mirrors
+- supporters
+author: dduportal
+opengraph:
+  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
+description: >
+  We are grateful for all the support from our supporters providing mirrors for Jenkins downloads.
+---
+
+The Jenkins project artefacts are provided through download mirror servers provided by organization supporting the project.
+
+You can see the list from the link:https://get.jenkins.io/index.html?mirrorstats[mirror status page].
+
+We're really grateful for all the support provided by the organization behind the curtain, namely:
+
+- link:https://osuosl.org[The Oregon State University Open Source Lab] in USA
+- link:https://www.tsinghua.edu.cn[The Tsinghua University] in China
+- link:https://xmission.com[XMission] in USA
+- link:https://belnet.be/[Belnet] in Belgium
+- link:https://www.rwth-aachen.de/[The RWTH Aachen University] in Germany
```

Comment:
I'd leave out "The" (also for the other universities), just "RWTH Aachen University" is fine

Reply:
```suggestion
- link:https://www.rwth-aachen.de/[RWTH Aachen University] in Germany
```